docs/UserGuide.md

```diff
@@ -3,14 +3,141 @@ layout: page
+* [Getting Started](#getting-started)
+  * [Installing and launching TaskHub](#installing-and-launching-taskhub)
+  * [Understanding the components of TaskHub](#understanding-the-components-of-taskhub)
+  * [Understanding the User Interface(UI)](#understanding-the-user-interface--ui-)
+  * [Quick start](#quick-start)
+* [Features](#features)
+  * [General Commands](#general-commands)
+    * [View help : `help`](#view-help--help)
+      * [GUI Help:](#gui-help-)
+      * [CLI Help](#cli-help)
+    * [List all employees and projects: `list`](#list-all-employees-and-projects--list)
+    * [Clear all entries : `clear`](#clear-all-entries--clear)
+    * [Exit the program : `exit`](#exit-the-program--exit)
+  * [Employee Commands](#employee-commands)
+    * [Add an employee: `addE`](#add-an-employee--adde)
+    * [Edit an employee : `editE`](#edit-an-employee--edite)
+    * [Delete an employee : `deleteE`](#delete-an-employee--deletee)
+    * [List all employees : `listE`](#list-all-employees--liste)
+    * [Locate employees by name: `findE`](#locate-employees-by-name--finde)
+  * [Project Commands](#project-commands)
+    * [Add a new project : `addP`](#add-a-new-project--addp)
+    * [Edit a project : `editP`](#edit-a-project--editp)
+    * [Delete a project: `deleteP`](#delete-a-project--deletep)
+    * [Mark project(s) as completed: `markP`](#mark-project--s--as-completed--markp)
+    * [Mark project(s) as incomplete: `unmarkP`](#mark-project--s--as-incomplete--unmarkp)
+    * [Edit deadline of (a) project(s): `dlP`](#edit-deadline-of--a--project--s---dlp)
+    * [Prioritise projects: `priorityP`](#prioritise-projects--priorityp)
+    * [List all projects: `listP`](#list-all-projects--listp)
+    * [Locate projects by name: `findP`](#locate-projects-by-name--findp)
+  * [Task Commands](#task-commands)
+    * [Add a new task to a project: `addT`](#add-a-new-task-to-a-project--addt)
+    * [Delete a task from a project: `deleteT`](#delete-a-task-from-a-project--deletet)
+    * [Mark task(s) as completed: `markT`](#mark-task--s--as-completed--markt)
+    * [Mark task(s) as incomplete: `unmarkT`](#mark-task--s--as-incomplete--unmarkt)
+    * [Sort tasks by deadline and completion status: `sortT`](#sort-tasks-by-deadline-and-completion-status--sortt)
+  * [Assignment Commands](#assignment-commands)
+    * [Assign employee(s) to a project: `assignP`](#assign-employee--s--to-a-project--assignp)
+    * [Un-assign employee(s) from a project: `unassignP`](#un-assign-employee--s--from-a-project--unassignp)
+    * [Assign an employee to a task: `assignT`](#assign-an-employee-to-a-task--assignt)
+    * [Un-assign an employee from a task: `unassignT`](#un-assign-an-employee-from-a-task--unassignt)
+  * [Data Storage](#data-storage)
+    * [Save the data](#save-the-data)
+    * [Edit the data file](#edit-the-data-file)
+  * [FAQ](#faq)
+  * [Known issues](#known-issues)
+  * [Command summary](#command-summary)
```

Comment:
Think the above 4 sections should be indented once to the left

Reply:
Yep! Will fix that